ansible/roles/pmm2-images/tasks/main.yml

```
@@ -36,10 +36,18 @@
           - python2-pip
 
     - name: Create pmm user           | Create pmm user
-      when: ansible_virtualization_type != "docker"
       user:
-        name: pmm
-        comment: PMM Server
+        name: "{{ item.name }}"
+        uid: "{{ item.uid }}"
+        home: "{{ item.home }}"
+        comment: "{{ item.comment }}"
+        shell: "{{ item.shell }}"
+      loop:
+        - { name: pmm, uid: 1000, comment: "PMM Server", shell: "/bin/false", home: "/home/pmm" }
+        - { name: nginx, uid: 999, comment: "nginx user", shell: "/sbin/nologin", home: "/var/cache/nginx" }
```

Let's try `/var/lib/nginx`, `cache` is probably not the right directory for home
```suggestion
        - { name: nginx, uid: 999, comment: "nginx user", shell: "/sbin/nologin", home: "/var/lib/nginx" }
```

---

It's how it works in nginx packages :) 
https://github.com/easel/nginx-centos-rpm/blob/master/nginx.spec#L3

---

Yes, and this is exactly `/var/lib/nginx`, not cache